.github/workflows/ci-latest-release.yml

```diff
@@ -101,8 +123,8 @@ jobs:
 
   push-stable-version:
     name: Create KubeArmor stable release
-    needs: build
-    if: github.ref != 'refs/heads/main'
+    needs: [build, check]
+    if: github.ref != 'refs/heads/main' && needs.check.outputs.kubearmor == 'true'
```

Comment:
push stable should not be limited to kubearmor, we need stable images for controller as well.


Noting need to add regctl copy for controller versioned release too